### DISCUSSION TITLE
技术预览版 - 3

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#194] [通知]
## 🐛 Bug 修复

- 修复收藏夹选中问题 @Richasy (#193)
- 修复小窗模式下的空隙 @Richasy (#185)
- 调整播放控件的最大宽度 @Richasy (#183)
- 完善评论展示 @Richasy (#175)
- 修复打开启动设置项会造成应用闪退的问题 @Richasy (#155)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2109.2001.pre-release'>技术预览版 - 3</a>.</em>

**Comment by LProphet** [comment]
牛的

**Comment by minortex** [comment]
幸苦辣，一天一版本

**Comment by sunxl3** [comment]
辛苦了

**Comment by mesondzh** [comment]
辛苦了

**Comment by Pairman** [comment]
辛苦！

**Comment by X0980** [comment]
牛蛙，辛苦了，不过云大佬是不是对那个媒体控件过宽的反馈有什么误解？嘿嘿
